plugins/notion/src/SelectDatabase.tsx

```diff
@@ -55,39 +73,26 @@ export function SelectDatabase({ onDatabaseSelected }: SelectDatabaseProps) {
                 <div className="flex gap-[10px]">
                     <select
                         value={selectedDatabase ?? ""}
-                        onChange={e => setSelectedDatabase(e.target.value)}
+                        onChange={event => setSelectedDatabase(event.target.value)}
                         className="flex-1 shrink-1"
-                        disabled={data && data.length === 0}
+                        disabled={!selectEnabled}
                     >
-                        {isLoading && (
-                            <option disabled selected>
-                                Loading…
-                            </option>
```

Comment:
I saw warnings in the console that you should not make select options selected like this. It already automatically gets selected so not a problem I think